tests/damage_rheology_latent_heat.prm

```diff
@@ -0,0 +1,138 @@
+#########################################################
+# This is a model setup to test the latent heat generation. 
+# It is identical to the latent_heat_enthalpy test, but utilizes
+# the damage rheology material model to ensure a correct 
+# implementation of the functionality. For details on the 
+# test see the latent_heat_enthalpy testcase.
+
+set Dimension = 2
+
+set Start time                             = 0
+set End time                               = 1e15
+set Use years in output instead of seconds = false
+set Adiabatic surface temperature          = 1000
+
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X extent = 50000
+    set Y extent = 1000000
+    set Y repetitions = 20
```

Comment:
This test has a different geometry (and refinement level) than the `latent_heat_enthalpy` test, which is the only thing we have to compare it with. They should be made consistent, so that we can then see if their results are identical (which they should be, I think).